core/src/main/java/org/elasticsearch/discovery/zen/ping/unicast/UnicastZenPing.java

```diff
@@ -64,7 +64,7 @@
 
     public static final String ACTION_NAME = "internal:discovery/zen/unicast";
 
-    public static final int LIMIT_PORTS_COUNT = 1;
+    public static final int LIMIT_PORTS_COUNT = 10;
```

Comment:
Maybe when its localhost we dont need any limiting, not sure, we should look into it. At least any limiting should be "per-address", since a host here could resolve to multiple addresses (e.g. ipv4 and v6, among other possibilities).